packages/features/bookings/Booker/Booker.tsx

```diff
@@ -52,6 +57,14 @@ const BookerComponent = ({
   const extraDays = isTablet ? extraDaysConfig[layout].tablet : extraDaysConfig[layout].desktop;
   const bookerLayouts = event.data?.profile?.bookerLayouts || defaultBookerLayoutSettings;
   const animationScope = useBookerResizeAnimation(layout, bookerState);
+  const isEmbed = typeof window !== "undefined" && window?.isEmbed?.();
+  const isMonthView = layout === BookerLayouts.MONTH_VIEW;
+  // We only want the initial url value, that's why we memo it. The embed seems to change the url, which sometimes drops
+  // the layout query param.
+  const layoutFromQueryParam = useMemo(() => validateLayout(getQueryParam("layout") as BookerLayouts), []);
+  const defaultLayout = isEmbed
```

Comment:
Embed won't use event settings, but rather will take the layout from the query param. That's set in the settings of the embed.